clang/test/CodeGenOpenCL/preserve_vec3.cl

```diff
@@ -1,77 +0,0 @@
-// RUN: %clang_cc1 %s -emit-llvm -o - -triple spir-unknown-unknown -fpreserve-vec3-type | FileCheck %s
-
-typedef char char3 __attribute__((ext_vector_type(3)));
```

Comment:
Shouldn't delete the test

Reply:
The point of the test is for preserving vec3, as the name of the file indicates, which is no longer the case. Why do we want to keep it?

Reply:
I'm not sure we have adequate coverage of the 3 element vector types elsewhere. You could remove the flag